From 00703f1110cd79ec8865cc60a5dd50d43a03ef73 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Wed, 16 Nov 2022 11:34:15 +0200
Subject: [PATCH] [SW-66407] gaudi3: pcie_fw: adjust firmware size

Pcie Phy fw binary image size is adjusted according to the content
of the provided image.
The provided hex file is actually has 64Kb of data, but the end of
the file is filled with zeroes, and there is no need to waste flash
access time for that.

Change-Id: Ic733ea954cb89b9ed306365c607cfa9bcb8404f3
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/Makefile b/arch/gaudi3/arc/Makefile
index 061f56e..8487857 100644
--- a/arch/gaudi3/arc/Makefile
+++ b/arch/gaudi3/arc/Makefile
@@ -1,6 +1,6 @@
 include $(HL_DEV_ARCH_COMMON)/Makefile
 
-PHY_FW_SIZE := 0x8000
+PHY_FW_SIZE := 0xA500
 PPBOOT_MAX_SIZE := 16384
 PREBOOT_BIN_MAX_SIZE := 130048
 
